apps/script/deploy-storybook.sh

```diff
@@ -19,8 +19,8 @@ rm -rf $DIR_TO_DEPLOY
 echo "Compiling application.css using rails asset pipeline"
 pushd ../dashboard
 rails assets:precompile
-cp public/assets/css/application-*.css ../apps/build/package/application.css || :
-cp public/assets/application-*.css ../apps/build/package/application.css || :
+cp public/assets/css/application-*.css ../apps/build/package/application.css 2>/dev/null || :
+cp public/assets/application-*.css ../apps/build/package/application.css 2>/dev/null || :
```

Comment:
Wait, I don't understand... if these are going to fail, why don't we just remove them?  Or are we expecting at least one of these to succeed - and if so, can we check for that?  I thought they both failed in the log I sent you (but can't check right now due to the S3 outage).